tests/executables_src/testScriptedInitialisationHandler.cc

```diff
@@ -107,27 +108,18 @@ BOOST_FIXTURE_TEST_CASE(testError, Fixture) {
   // let the script run three times. Afterwards manually check that there is only one printout on the console
```

Comment:
The comment does not match any more. It checks that the message changes if the script output changes.

Reply:
fixed